kclvm/parser/src/parser/stmt.rs

```diff
@@ -737,7 +737,14 @@ impl<'a> Parser<'a> {
         if let TokenKind::Indent(VALID_SPACES_LENGTH) = self.token.kind {
             let body = self.parse_schema_body();
 
-            let pos = self.token_span_pos(token, self.prev_token);
+            let mut pos = self.token_span_pos(token, self.prev_token);
```

Comment:
CI and DCO failed. Besides, you can use `fix #1099` in the description of PR to automatically close the issue when the PR is merged.